src/lib/requests.ts

```diff
@@ -32,9 +32,11 @@
 /* istanbul ignore file */
 // Testing will be covered in #354
 
-import Logger from '@mojaloop/central-services-logger'
 import { config } from './config'
-import { ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import Logger, {
```

Comment:
Just checking, `sdk-standard-components` has `Logger` as a default import?

Reply:
Yes @stevenwjy . Both `central-services` and `sdk-standard-components` have `Logger`s.